5watchdog

```diff
@@ -167,16 +167,16 @@ sleep 10
 
 REUP_MINER=0
 while ! ps ax | grep SCREEN | grep -v cpuminer | grep -q miner; do
-  echo "$(date) - [WARNING] - Miner not running, commencing miner start command again" 
-  bash ${NVOC}/0miner
+  echo "$(date) - [WARNING] - Miner not running, reinit nvOC from scratch" 
+  pkill -f 3main
```

Comment:
Restarting 3main will not launch miner as fast as commencing 0miner and this will cause a 3main start/kill loop.
This code need a total revise to work without causing problem, or remove and rely on low utilization to catch no miner running.

Reply:
It has to work with salfter also, so launching 0miner is not an option unless we create different cases --  not the best I can think to…

The point is: if 3main failed to launch the miner, why starting it here manually should work? Which case this code was intended to address?

Reply:
Dirty workaround ...  terminate watchdog by "pkill -f 5watchdog"  after "pkill -f 3main" and let it start again by 3main?

Reply:
I'm unable to imagine a startup sequence which could result in a miner first-launch failure, and an immediate next attempt to succeed. Let's assume there are issues in user settings in 1bash, then restarting only the miner is not enough even if meanwhile settings get fixed. I think continuing to kill and restart 3main until its miner stay up MAY be fine. The watchdog itself does not need to be killed since it is not relaunched if is up already.

Reply:
This code is not to solve any issues, but to prevent further problems, for example I make a mistake in 1bash and restart 3main, miner exits and watchdog will restart 3main 4 times and eventually reboot the rig. 
Problem with restart 3main and not terminate watchdog is here:
https://github.com/papampi/nvOC_by_fullzero_Community_Release/blob/4852aadad539cade94efd61d4fb6fe1a5296df44/5watchdog#L171-L172
It will kill 3main and check for running miner right after that, while we know it will take much longer for 3main get to start the miner.
Lets remove this miner check before loop starts till we find a better solution for it.

Reply:
That line is not a check, was just to print the miner command it detected as running. I just removed it. See if it could be ok now, otherwise I'll drop that check. There will be a 10 seconds timeout for 3main to restart the miner, is it enough?

Reply:
I agree, salfter's switchers are still applying oc one parameter at a time, I will change them either.

Reply:
Dont think so many users uses it
All the time since 19-2.0 no one reported the fan problem.

Reply:
or the myr_gr or ...

Reply:
Workaround is complete, will do a reasonable watchdog rework when we find a way to avoid tons of nvidia-smi queries